### DISCUSSION TITLE
The first time I've seen such an error message, I can't locate where the error occurred. Any clues?

### DISCUSSION THREAD

**Discussion by IronEaterBeast** [AvaloniaUI/Avalonia#14278] [Q&A] [answered]
[Binding]Error in binding to 'Avalonia.Input.KeyBinding'.'Command': 'Null value in expression '{empty}' at ''.'(KeyBinding #59763787)
[Binding]Error in binding to 'Avalonia.Controls.RowDefinition'.'Height': 'Null value in expression '{empty}' at ''.'(RowDefinition #1003178)
[Binding]Error in binding to 'Avalonia.Controls.RowDefinition'.'Height': 'Null value in expression '{empty}' at ''.'(RowDefinition #9028608)
“System.ArgumentNullException”(Avalonia.Controls.dll )

**Comment by thevortexcloud** [comment]
Is this an error in the previewer or at runtime?

**Comment by IronEaterBeast** [comment]
> Is this an error in the previewer or at runtime?

Runtime, debug.

**Reply by thevortexcloud** [reply, marked as answer]
If you are using Visual Studio, make sure you have enabled `ArgumentNullException` in your exception settings to have it automatically stop at the exception. It might give you a call stack/stop at the exact line that is causing the issue which may give you a clue.

**Reply by timunie** [reply]
If App doesn't crash ignore these logs for now. It should be fixed in 11.1 hopefully

**Reply by IronEaterBeast** [reply]
Ok, thank you.
